app/tests/tests_10/test_asset_upload_endpoint.py

```diff
@@ -497,6 +501,8 @@ def test_asset_upload_gzip(self):
         self.assertS3ObjectExists(key)
         obj = self.get_s3_object(key)
         self.assertS3ObjectContentEncoding(obj, key, encoding='gzip')
+        self.asset.refresh_from_db()
+        self.assertEqual(size, self.asset.file_size)
```

Comment:
There seems to be a slight discrepancy in the file size in this case. I'm not sure if this is expected behavior or there is an error in the code.